### ISSUE TITLE
In train_RINDNet.py, size mismatch error

### ISSUE THREAD

**Issue by carwyn987** [MengyangPu/RINDNet#12]
Line 89:

`
loss = self.criterion([out_depth,out_normal,out_reflectance,out_illumination], target[:, 1:5, :, :])
`

 I get the following error:

_ValueError: Using a target size (torch.Size([4, 320, 320])) that is different to the input size (torch.Size([5, 1, 320, 320])) is deprecated. Please ensure they have the same size._

Fixed by replacing line with: 

`
loss = self.criterion(torch.cat((out_depth,out_normal,out_reflectance,out_illumination),axis=1), target[:, 1:5, :, :])
`

Unsure if this fix breaks training, but fixes error and allows training to commence.

Using Python 3.7, numpy 1.19.5, torch 1.13.1, opencv-python 4.3.0.38, ...

**Comment by MengyangPu** [comment]
Thanks, this fix will not affect training, or replacing line with:
`loss = self.criterion(torch.cat((out_depth,out_normal,out_reflectance,out_illumination),dim=1), target[:, 1:5, :, :])`
